services/apis/pidgin/pidginTasks.js

```diff
@@ -5,19 +5,7 @@ let I = actor();
  * pidgin Tasks
  */
 module.exports = {
-  //API Example:
-  getFiles() {
-    I.sendGetRequest(sheepdog.endpoints.getFile, accessTokenHeaders)
-  },
   
-  //Portal Example:
-  goTo() {
-    homepage_service.do.goTo();
-    portalUtil.clickProp(navbar.props.dictionary_link)
-    portalUtil.seeProp(dictionaryProps.ready_cue)
-  },
-  
-  //FROM ORIGINAL PIDGIN STEPS:
   util : require('./utilSteps'),
   accessTokenHeader : util.getAccessTokenHeader(),
```

Comment:
this util file does not exist anymore - you have to use the "user" module to get a token. for example https://github.com/uc-cdis/gen3-qa/blob/8f5c1e35fc306ba6cdc324ea1851df63b1999bc4/suites/portal/dataUploadTest.js#L86